### PR TITLE
Implement app content

### PR DESCRIPTION
**Pull request by dmrib**
Depends on #2 

First commit is: `Implement content section stylesheet`

### CODE REVIEW

**Review by HeitorBoschirolli — COMMENTED** [review]
no version update commit?

File: linguicator-editor/src/components/Content/Content.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Content.css'
+
+
+function Content()
```
Comment (HeitorBoschirolli) [inline linguicator-editor/src/components/Content/Content.js]: Describe this component if we decide to set is as a guideline

**Review by HeitorBoschirolli — CHANGES_REQUESTED** [review]
Nothing different from the previous MR. Version update commit is also missing btw

File: linguicator-editor/src/components/Content/Content.js (hunk 1 above)
Comment (dmrib) [reply linguicator-editor/src/components/Content/Content.js]: Done!

File: src/App.js [hunk 2]
```diff
@@ -11,7 +12,10 @@ import 'antd/dist/antd.css';
  */
 function App() {
   return (
-      <Header />
+    <div className='app-container'>
+          <Header />
```
Comment (HeitorBoschirolli) [inline src/App.js]: The indentation in component files will be two or four spaces? It's not consistent in this file.

Comment (dmrib) [reply src/App.js]: I updated the entire project to use 4.

**Review by HeitorBoschirolli — APPROVED** [review]
+1

**Pull request merged**